evm/its.js

```diff
@@ -426,50 +418,49 @@ async function processCommand(config, chain, options) {
             break;
         }
 
-        case 'setTrustedAddress': {
+        case 'set-trusted-chains': {
+            const [itsChain, itsAddress] = args;
             const owner = await new Contract(interchainTokenService.address, IOwnable.abi, wallet).owner();
 
             if (owner.toLowerCase() !== walletAddress.toLowerCase()) {
                 throw new Error(`${action} can only be performed by contract owner: ${owner}`);
             }
 
-            validateParameters({ isNonEmptyString: { trustedChain: options.trustedChain } });
+            validateParameters({ isNonEmptyString: { itsChain } });
 
             let trustedChains, trustedAddresses;
 
-            if (options.trustedChain === 'all') {
-                trustedChains = parseTrustedChains(config, options.trustedChain);
-                trustedAddresses = trustedChains.map((_) => options.trustedAddress || chain.contracts?.InterchainTokenService?.address);
+            if (itsChain === 'all') {
+                trustedChains = parseTrustedChains(config, itsChain);
```

Comment:
```suggestion
                trustedChains = parseTrustedChains(config, [itsChain]);
```
shouldn't this take a list?